packages/backend-lib/src/journeys/userWorkflow/activities.ts

```diff
@@ -677,6 +678,54 @@ async function sendEmailWithPayload(
             replyTo,
           });
         }
+        case EmailProviderType.Resend: {
```

Comment:
This is actually deprecated code. Feel free to replace this with a simple `throw` statement here. It'll be deprecated soon.